roles/rsyslog/roles/input_roles/ovirt/tasks/main.yaml

```diff
@@ -1,12 +1,12 @@
 ---
 - name: Set oVirt facts
   set_fact:
-    rsyslog_role_packages: "{{ rsyslog_ovirt_prereq_packages|flatten|d([]) }}+{{ rsyslog_ovirt_packages|flatten|d([]) }}"
+    rsyslog_role_packages: "{{ rsyslog_ovirt_prereq_packages | d([]) + rsyslog_ovirt_packages | d([]) }}"
```

Comment:
See above - can it be renamed to `__rsyslog_packages` and moved to vars, eliminating `rsyslog_ovirt_packages` ?
What will happen if one uses both the `basics` and `ovirt` subroles? Won't the second `set_fact` clobber the result of the former? is that expected?

Reply:
maybe I misunderstood - I see there are `rsyslog_base_packages` and `rsyslog_role_packages` variables with different functions - why is it the case?

Reply:
The rsyslog_role_packages are taken care in roles/rsyslog/tasks/deploy.yaml vir each deploy.yaml following tasks_from line (e.g., https://github.com/linux-system-roles/logging/pull/87/files#diff-32e44b317e7ab0e0016656a25254fa0bL10, here).  So, ovirt packages are installed immediately after this set_fact and the same happens for basics, too.

> Won't the second set_fact clobber the result of the former?

So, the answer is "no".  It does not clobber the result of the former, but the value of rsyslog_role_packages is used immediately before being clobbered.

Reply:
https://github.com/linux-system-roles/logging/pull/87#discussion_r381552024 is applied here, as well.

Reply:
Note: var name `rsyslog_role_packages` was renamed to `__rsyslog_packages` and the var is set with instead of `set_fact` but in `var:`.